lib/SPIRV/OCLToSPIRV.cpp

```diff
@@ -812,7 +812,8 @@ void OCLToSPIRVBase::visitCallGroupBuiltin(CallInst *CI,
                                   .Case("ballot_exclusive_scan", "add")
                                   .Default(FuncName.take_back(
                                       3)); // assumes op is three characters
-          GroupOp.consume_front("_");      // when op is two characters
+          if (!GroupOp.consume_front("_")) // when op is two characters
+            return true;                   // continue
```

Comment:
This seems incorrect. Shouldn't it be:
```suggestion
          (void)GroupOp.consume_front("_")); // when op is two characters
```

Reply:
Ah...I think you are right. We can ignore the result here. Good catch.

Thanks